readthedocs/doc_builder/python_environments.py

```diff
@@ -187,7 +187,7 @@ def install_core_requirements(self):
 
         # Use conda for requirements it packages
         requirements = [
-            'sphinx==1.3.1',
+            'sphinx==1.3.5',
```

Comment:
Wee, yeah was going to raise this question after local testing. I assumed there was a reason it was pinned at 1.3.1